docs/docs/usage/home-assistant.md

```diff
@@ -131,3 +131,9 @@ automation:
               - url: 'http://ccab4aaf-frigate:5000/api/events/{{trigger.payload_json["after"]["id"]}}/thumbnail.jpg'
                 caption: 'A {{trigger.payload_json["after"]["label"]}} was detected on {{ trigger.payload_json["after"]["camera"] }} camera'
 ```
+
+## FAQ
+
+### When using the [HomeKit integration](https://www.home-assistant.io/integrations/homekit/) in Home Assistant to expose cameras to HomeKit, the camera's motion sensor in HomeKit do not appear to function properly.
```

Comment:
This is not actually a question (as in FA**Q**) :-) How about:

If I am detecting multiple objects, how do I assign the correct `binary_sensor` to the camera in HomeKit?

Reply:
Lol, I figured someone would say that. ;) I was thinking about just titling it “Troubleshooting & FAQ” like [this page](https://blakeblackshear.github.io/frigate/troubleshooting), but anything works fine for me. I was just trying to lead a person down the path I went down - noticed a motion sensor in HomeKit but couldn’t figure out why it wouldn’t trigger.

Reply:
Personally, I prefer FAQs to only be literal questions. I'd also prefer a neutral question (e.g. my alternative) rather than a negative (i.e. something appears broken) -- gotta be optimistic in open source ;-)